src/backgrounds/StarryBackground.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @overview A module that initializes the Starry Background parallax effect
+ * @module StarryBackground.js
+*/
+
+export const StarryBackground = {
+
+  /**
+   * Finds page modules with the Starry Background selector and triggers setup
+   * @returns {void}
+  */
+  init() {
+    this.StarryBackgroundEls = document.querySelectorAll(`.background--starry`);
```

Comment:
Should this be a JS class or is this an exception? If this an exception, then what's the rule...is it because the JS is really just applying styles? 

Or could we use data attributes? What do we think of that alternative `data-background="starry"? Would we extend the idea of data attributes for backgrounds to all our background patterns?